config.yml

```diff
@@ -78,8 +78,8 @@ menu:
     - identifier: teams
       name: Teams
       weight: 7
-    - identifier: billing
-      name: Billing
+    - identifier: billing-pricing
```

Comment:
We shouldn't change the identifier that easily, it will break links from the app. We could use page aliases but I think in this case it makes sense to only change the name of the category and not the name of the folder.

Reply:
reverted to original name